fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -458,7 +458,7 @@ public void execute() throws Exception {
 
         // Only add the last running stmt for multi statement,
         // because the audit log will only show the last stmt.
-        if (context.getIsLastStmt()) {
+        if (context.getIsLastStmt() && parsedStmt != null) {
```

Comment:
may necessary add comments to explain when this parsedStmt can be null. the `parsedStmt` is rarely checked if null or not explicitly in this StmtExecutor class.